searchlib/src/vespa/searchlib/query/tree/intermediate.cpp

```diff
@@ -9,10 +9,29 @@ Intermediate::~Intermediate() {
     }
 }
 
-Intermediate &Intermediate::append(Node::UP child)
+Intermediate &
+Intermediate::append(Node::UP child)
 {
     _children.push_back(child.release());
     return *this;
 }
 
+Intermediate &
+Intermediate::prepend(Node::UP child)
+{
+    _children.insert(_children.begin(), child.release());
```

Comment:
not as exception safe as it might look, but I assume we always crash on all allocation failures anyway, so this should be a non-issue.

Reply:
Agree.